libraries/AP_SmartRTL/AP_SmartRTL.cpp

```diff
@@ -330,7 +330,7 @@ void AP_SmartRTL::run_background_cleanup()
     const uint16_t path_points_completed_limit = _path_points_completed_limit;
     _path_points_completed_limit = SMARTRTL_POINTS_MAX;
     _path_sem.give();
-
```

Comment:
slight issue with the formatting as well ([style guide](http://ardupilot.org/dev/docs/style-guide.html)).  We always use curly brackets even if there's only a single statement afterwards.

Reply:
was wondering about that....will fix

Reply:
fixed